sorting/sorting.go

```diff
@@ -0,0 +1,123 @@
+package sorting
+
+import (
+	"bufio"
+	"errors"
+	"log"
+	"os"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// a defined type to simulate a tuple with three elements as sepecified in the associated function
+type Tuple []int
+
+func (tuple *Tuple) appendOne(number int) error {
+	/* This function appends one number to the defined type Tuple if it's not already full (with 3 numbers)
+	otherwise it'll return an error that will be logged using the function HandleErrors */
+	if len(*tuple) > 3 {
+		return errors.New("cannot add anymore numbers")
+	}
+	*tuple = append(*tuple, number)
+	return nil
+}
+
+func ReadingTuplesFromFile(fileName string) []Tuple {
```

Comment:
Minor: Exported functions in go require comments. You get lint warning otherwise.
Check **Funcs** section in https://tip.golang.org/doc/comment